database_scheme.sql

```diff
@@ -10,10 +10,20 @@ CREATE TABLE users
     is_admin    BOOLEAN NOT NULL DEFAULT FALSE
 );
 
+/*
+ Note that unlike `send_invitations` table, `conversations` stores information about **requested conversations**, not
+ invitations messages.
+
+ `conversations.operator_chat_id` set to `NULL` indicates that the user is waiting has requested a conversation.
+ Information of this kind is stored in the `conversations` table in order to have current conversations and conversation
```

Comment:
Not accurate enough, too
```suggestion
 Information about requests is stored in the `conversations` table, not in a separate one, to have current conversations and conversation
```